src/routes/interested/index.js

```diff
@@ -50,16 +50,28 @@ export default class Interested extends Component {
   );
 
   renderPlaceRegistration = placeNumber => (
-    <div className={style.content}>
-      <Heading>Zarezerwowano miejsce:</Heading>
-      <PlaceNumber reserved number={placeNumber} />
-      <Button
-        Primary
-        href='/'
-        onClick={this.releasePlace}
-      >
-        Zwolnij
-      </Button>
+    <div className={`${style.content} ${style.spaceBetween}`}>
```

Comment:
rather than this inline strings try to use `classnames`

Reply:
👍